src/Sentry.Unity.Editor.iOS/MainModifier.cs

```diff
@@ -0,0 +1,71 @@
+using System.IO;
+using System.Text.RegularExpressions;
+using UnityEngine;
+
+namespace Sentry.Unity.Editor.iOS
+{
+    public static class MainModifier
+    {
+        private const string Include = "#include <Sentry/Sentry.h>\n#include \"SentryOptions.m\"\n";
+        private const string Init = "\t\t[SentrySDK startWithOptions:getSentryOptions()];\n\n";
```

Comment:
What about (to replace the tabs with spaces):

```suggestion
        private const string Init = @"
        [SentrySDK startWithOptions:getSentryOptions()];

";
```